resources/assets/less/app.less

```diff
@@ -384,6 +384,23 @@ a.logo.no-hover a:hover {
   background-color: transparent;
 }
 
+.index-blocks {
+  color: var(--text-sub);
```

Comment:
This makes the text a lot lighter for Default Blue (at least). Was this added to handle something for dark mode?
![image](https://github.com/user-attachments/assets/56d6324a-f439-4518-bf24-7a6188ecece2)
vs
![image](https://github.com/user-attachments/assets/229f0ab1-e950-4a98-9d84-2f2bbc5224cb)

Reply:
yea, so iirc the text i think is lighter, so when you highlight it, you see the full text and it kinda lights up a bit. would be a thing on dark themes, but yea looks like it gets worse on light screens